runtime/pallene_core.h

```diff
@@ -226,4 +226,41 @@ int pallene_is_truthy(const TValue *v)
     return !(ttisnil(v) || (ttisboolean(v) && bvalue(v) == 0));
 }
 
+static inline
+Table *pallene_new_table(lua_State *L, lua_Integer n)
+{
+    Table *t = luaH_new(L);
+    if (n > 0) {
+        luaH_resize(L, t, 0, n);
+    }
+    return t;
+}
+
+static const TValue PALLENE_ABSENTKEY = {ABSTKEYCONSTANT};
+
+static inline
+TValue *pallene_getstr(Table *t, TString *key, int *pos)
+{
```

Comment:
Apparently there are two getstr's in Lua. One is a macro defined in lobject.h and one is a function from ltable.c. Maybe we should add a comment to clarify this is referring to luaH_getstr from ltable.c.